pool-agent/cmd/create.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"fmt"
+	"log/slog"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/lxc/lxd/shared/api"
+)
+
+func (a *Agent) createInstance(name string, rt ResourceTypesMap, l *slog.Logger) error {
+	l.Info("Creating instance")
```

Comment:
Please set the instance name in the logger for operation.

Reply:
Instance name has already set by caller.
https://github.com/gamoutatsumi/shoes-lxd-multi/blob/39cb6bfe6a08ef177c2aafd4f9ee3742545a43a7/pool-agent/cmd/agent.go#L206